--- logjam-4.4.1/src/tools.c.BAD	2005-03-13 17:17:31.000000000 -0600
+++ logjam-4.4.1/src/tools.c	2005-03-13 17:54:53.000000000 -0600
@@ -209,7 +209,7 @@
 
 	while (gtk_dialog_run(GTK_DIALOG(cmd_dlg)) == GTK_RESPONSE_OK) {
 		const gchar *command;
-		const gchar *encoding;
+		const gchar *encoding = 0;
 		GError *err = NULL;
 
 		command = gtk_entry_get_text(GTK_ENTRY(entry));
--- logjam-4.4.1/src/checkfriends-gtk.c.BAD	2005-03-13 17:50:06.000000000 -0600
+++ logjam-4.4.1/src/checkfriends-gtk.c	2005-03-13 17:51:41.000000000 -0600
@@ -255,8 +255,6 @@
 
 static void
 cf_float_destroy(CFFloat *cff) {
-	CFMgr *cfm = cff->cfmgr;
-
 	if (GTK_IS_WIDGET(cff->win))
 		gtk_widget_destroy(cff->win);
 
@@ -277,8 +275,6 @@
 #ifdef USE_DOCK
 static void
 cf_dock_destroy(CFFloat *cff) {
-	CFMgr *cfm = cff->cfmgr;
-
 	g_signal_handlers_disconnect_by_func(G_OBJECT(cff->win),
 			G_CALLBACK(cf_dock_destroyed_cb), NULL);
 	gtk_widget_destroy(GTK_WIDGET(cff->win));
@@ -290,7 +286,11 @@
 cf_dock_destroyed_cb(GtkWidget *widget, CFFloat *cff) {
 	app.docklet = NULL;
 }
-	
+
+/* 
+ * Not currently used.	
+ */
+/*
 static void
 cf_dock_setup(CFFloat *ind) {
 	GtkWidget *docklet = app.docklet;
@@ -303,6 +303,7 @@
 
 	ind->win = docklet;
 }
+*/
 #endif /* USE_DOCK */
 
 void cf_float_decorate_refresh(void) {
--- logjam-4.4.1/src/cmdline.c.BAD	2005-03-13 17:41:34.000000000 -0600
+++ logjam-4.4.1/src/cmdline.c	2005-03-13 17:46:22.000000000 -0600
@@ -664,7 +664,7 @@
 
 static void
 command_dispatch(Cmdline *cmdline, Command *commands, const char *help, int argc, gchar *argv[]) {
-	JamAccount *acc;
+	JamAccount *acc = NULL;
 	Command *command = NULL;
 	char *cmdname;
 	int i;
--- logjam-4.4.1/src/menu.c.BAD	2005-03-13 17:52:10.000000000 -0600
+++ logjam-4.4.1/src/menu.c	2005-03-13 17:52:16.000000000 -0600
@@ -440,7 +440,6 @@
 
 };
 	int itemcount = sizeof(menu_items) / sizeof(menu_items[0]);
-	int i;
 	JamView *view;
 
 	accelgroup = gtk_accel_group_new();
