--- logjam-4.4.1/protocol/liblivejournal/livejournal/entry.c.BAD	2005-02-10 23:19:30.000000000 -0600
+++ logjam-4.4.1/protocol/liblivejournal/livejournal/entry.c	2005-02-10 23:20:38.000000000 -0600
@@ -590,6 +590,12 @@ rfc822_load_entry(const char *key, const
 	else if (g_ascii_strcasecmp(key, "time") == 0) {
 		if (entry) lj_ljdate_to_tm(val, &entry->time);
 	}
+	else if (g_ascii_strcasecmp(key, "backdated") == 0) {
+		if (entry && val[0]) {
+			if (g_ascii_strcasecmp(val, "yes") == 0)
+				entry->backdated = TRUE;
+		}
+	}
 	else return FALSE;
 
 	return TRUE;
